cmd/tendermint/commands/testnet.go

```diff
@@ -50,6 +51,8 @@ func init() {
 		"Directory to store initialization data for the testnet")
 	TestnetFilesCmd.Flags().StringVar(&nodeDirPrefix, "node-dir-prefix", "node",
 		"Prefix the directory name for each node with (node results in node0, node1, ...)")
+	TestnetFilesCmd.Flags().Int64Var(&initialHeight, "initial-height", 0,
```

Comment:
Do we also want a flag added for the initializing an individual node (init.go)

Reply:
`tendermint init` doesn't appear to take any options at all (only global options), so I guess not? I suppose people configure the node by editing the genesis file?

Not to say we shouldn't add options, but in that case we should also add options for e.g. the chain ID and other settings. Not sure if this is needed.

Reply:
Yeah I guess leave it as is